Folder/email_notifier.py

```diff
@@ -0,0 +1,32 @@
+import os
+import smtplib
+import requests
+from dotenv import load_dotenv
+
+
+class EmailNotifier:
```

Comment:
You could add a comment explaining what `EmailNotifier` is used for. Comments help people read the code and better understand. Also even helps future you when you try to remember wtf you did 6 months ago

Reply:
yea generally code docs in python are are done with `"""` 

```suggestion
class EmailNotifier:
    """
    All things related to Email
    """
    
    def __init__ ......
```

Reply:
Should've let you guys know but I made updates and merged it already with my main (added docstrings, etc), but didn't make it into a pull request. Well it was one initially but I merged it like pretty immediately just cause I made most of the recommended corrections, just not the ones where I replied to Jon with a question n he hasn't responded